pom.xml

```diff
@@ -633,11 +633,6 @@ under the License.
               </goals>
               <id>default-test</id>
               <phase>test</phase>
-              <configuration>
-                <systemPropertyVariables>
-                  <multiZk>false</multiZk>
-                </systemPropertyVariables>
-              </configuration>
```

Comment:
If someone expands the test environments and adds another environment, based on pom.xml content alone he/she might add `<multiZk>false</multiZk>`, then the if statement in ZkTestBase.java will be incorrectly triggered. Using boolean for the if statement seems better to me.

Reply:
Good point! I'll add the check for the string value.